### DISCUSSION TITLE
String is not highlighted in TOML when on lhs of `=` operator

### DISCUSSION THREAD

**Discussion by venkata-j** [helix-editor/helix#11390] [Troubleshooting]
### Summary

Strings can be given on the left of `=` in a TOML file, but these arent highlighted like other strings in the file. Screenshot attached.
![image](https://github.com/user-attachments/assets/a06659cc-ce23-444f-bbd5-0fe8574ac507)

Does the TOML's `highlight.scm` have to be updated in the `runtime` directory? I can try and submit a patch if someone could tell me which files to look at and try to modify.

### Reproduction Steps

 

Write `"test" = "test"` in a TOML file, and the left will stay unhighlighted while the right will highlight

### Platform

Linux

### Terminal Emulator

gnome-terminal

### Installation Method

built from source from releases

### Helix Version

helix 24.7 (079f5442)

**Comment by the-mikedavis** [comment]
![toml](https://github.com/user-attachments/assets/56bcd8cc-f749-4fba-a917-624c3123edfd)

Quoted keys are themed and they're themed the same as unquoted keys rather than strings (since they are not strings). This is controlled by this part of the toml highlights: https://github.com/helix-editor/helix/blob/3fcf168c33fda76b4b611877db4ce43b3942f1ab/runtime/queries/toml/highlights.scm#L16-L20

And your theme's `"variable.other.member"` style

**Comment by venkata-j** [comment]
got it, thanks
